### ISSUE TITLE
Unable to authorize in DesktopEditors

### ISSUE THREAD

**Issue by 0xdod** [ONLYOFFICE/onlyoffice.github.io#119]
After registering the app on mendeley website, i get an error while trying to login and connect from Onlyoffice with mendeley.

```
This site can’t be reached
The webpage at https://api.mendeley.com/oauth/authorize?client_id=XXXX might be temporarily down or it may have moved permanently to a new web address.
ERR_UNSAFE_REDIRECT
```
I use the Onlyoffice Desktop editors v6.1

**Comment by St3v3n-4n4** [comment]
there is no solution yet i still have the same problem

**Comment by roks531** [comment]
still the same problem... Didnt find a solution

**Comment by garbunus** [comment]
Wow... same problem since 2021... ok. It never will be fixed...

**Comment by janxkoci** [comment]
Personally I stopped using Mendeley and moved to Zotero. I can no longer tolerate where Mendeley and Elsevier are heading.

**Comment by Kakarot5** [comment]
I have the same issue, entering the 'Application ID' works, and therefore opens the next menu. However, its on this menu that it gets stuck, when I press 'Log in' it takes me to the Mendeley Authorization page, I click proceed and the page then becomes unresponsive. I have tried this process many times and I always arrive at the same conclusion. I love OnlyOffice but without referencing, it has become pretty unusable for me.

**Comment by garbunus** [comment]
I have moved to Zotero and all good now!

---
*Eduardo Garbin*
Analista de Sistemas / Cientista de dados
Coordenador /  Consultor em ensaios clínicos
+55 61 99250 9351
***@***.***



Em qua., 18 de out. de 2023 às 16:39, AdrianAdamovic <
***@***.***> escreveu:

**Comment by GeoffClements** [comment]
I too moved to Zotero but this also fails to work for me. see #220. I guess no one else is seeing this?

**Comment by garbunus** [comment]
I see...
Zotero works well with libreoffice. Using .odt format.

Em sex., 20 de out. de 2023 06:59, Geoff Clements ***@***.***>
escreveu:

**Comment by ostriz2** [comment]
I am really interested about solution of this problem too, because I choose Onlyoffice insteod of Microsoft Office at my personal computer. But I have Word in my office at university and I am using Mendeley there. And it is better for me and my supervisor have the same programs...
Has anyone managed to get around it?

**Comment by GeoffClements** [comment]
> I see... Zotero works well with libreoffice. Using .odt format. 

Yes, I have this working. Sadly I need it to work with ONLYOffice due to a requirement for Word compatibility which ONLYOffice does better than LibreOffice.

**Comment by VladimirProkofiev52** [comment]
The Mendeley Plugin doesn't supported by Desktop Editors.
I've added your comments to  issue 48975.
You can contact [sales@onlyoffice.com](mailto:sales@onlyoffice.com) to discuss our developer's prioritites
